schema.json

```diff
@@ -0,0 +1,514 @@
+{
+  "$schema": "http://json-schema.org/LivingDocumentation",
```

Comment:
I would expect this to indicate the version of JSON schema that we expect, for example, if we want Draft 7:

    "$schema": "http://json-schema.org/draft-07/schema#",

Which version will want will perhaps depend a bit on which validation library we want to use (on either side) in #75.